### PR TITLE
Updated kit lists

### PR DESCRIPTION
**Pull request by scottclarke61**
I've tested my updates/edits on a release ([https://d2akpwi1y0kp37.cloudfront.net/]) and all looks good

### CODE REVIEW

**Comment by donal-s** [comment]
Updated this branch with the location in title change

File: src/model/DataDefinitions.ts [hunk 1]
```diff
@@ -1255,172 +1180,456 @@ export const A_LINE_BOX: ContainerTemplate = {
...
+      { ...MISSING_FIELDS, name: "i-gel", size: "2" },
+      { ...MISSING_FIELDS, name: "i-gel", size: "1.5" },
+      { ...MISSING_FIELDS, name: "i-gel", size: "1" },
+      { ...MISSING_FIELDS, name: "Broslow tape" },
+    ],
+  };
+
+export const RED_BAG_1: StorageAreaTemplate = {
+  storageAreaId: "red-bag-1",
+  name: "Red Bag 1",
```
Comment (donal-s) [inline src/model/DataDefinitions.ts]: Should this be "Medic 1 - Red Bag 1" ?

File: src/model/DataDefinitions.ts [hunk 2]
```diff
@@ -1255,172 +1180,456 @@ export const A_LINE_BOX: ContainerTemplate = {
...
+
+export const BLACK_BAG_RIGHT_POCKET: PartialFixedContainerTemplate = {
+  name: "Right Side Pocket",
+  containerType: "Pocket",
+  items: [{ ...MISSING_FIELDS, name: "Paeds BVM with clear mask" }],
+};
+
+export const BLACK_BAG_1: StorageAreaTemplate = {
+  storageAreaId: "black-bag-1",
+  name: "Black Bag (First Responder) 1",
```
Comment (donal-s) [inline src/model/DataDefinitions.ts]: Should this be "Medic 1 - Black Bag 1" ?

**Review by byronn-1 — APPROVED** [review]
lots of terminology I don't understand but looks ok to me

File: src/model/DataDefinitions.ts (hunk 2 above)
Comment (scottclarke61) [reply src/model/DataDefinitions.ts]: Yes, thanks - have renamed both

**Pull request merged**